sylvia-derive/src/contract/communication/wrapper_msg.rs

```diff
@@ -9,7 +9,8 @@ use syn::fold::Fold;
 use syn::spanned::Spanned;
 use syn::{Ident, ItemImpl, Type};
 
-/// Glue message is the message composing Exec/Query messages from several traits
+/// Glue message is the message composing Exec/Query/Sudo messages from several traits and a
```

Comment:
Adding a reference or more details here will be beneficial for newcomers. For e.g. some reasoning why sylvia generates additional wrapper messages, why it should be parsed in a different way etc.